edx_proctoring/views.py

```diff
@@ -900,7 +908,7 @@ def post(self, request, external_id):
         attempt = get_exam_attempt_by_external_id(external_id)
         if attempt is None:
             raise StudentExamAttemptDoesNotExistsException('not found')
-        self.make_review(attempt, request.data)
+        self.make_review(attempt, request.data, request)
```

Comment:
Yes, the request should be an optional parameter. Passing in the data makes testing easier (instead of having to create a fake request object for all of the tests)